iceoryx_posh/include/iceoryx_posh/internal/popo/receiver_port.hpp

```diff
@@ -59,9 +59,9 @@ class ReceiverPort : public BasePort
 
     // BEGIN REGION__APPLICATION // /* access from Application-------------------------------
 
-    void subscribe(const bool f_autoResubscribe = false,
-                   const uint32_t f_deliverySize = MAX_RECEIVER_QUEUE_CAPACITY); // deprecated
-    void subscribe(const uint32_t f_deliverySize = MAX_RECEIVER_QUEUE_CAPACITY);
+    virtual void subscribe(const bool f_autoResubscribe = false,
+                           const uint32_t f_deliverySize = MAX_RECEIVER_QUEUE_CAPACITY); // deprecated
```

Comment:
Please add the keyword `[[gnu::deprecated]`

Reply:
the whole file is deprecated and will be replaced within the next weeks